src/Components/Patient/UpdateStatusDialog.tsx

```diff
@@ -56,7 +52,7 @@ const resultTypes = [
   ...SAMPLE_TEST_RESULT,
 ];
 
-const roleStatusMap = { ...ROLE_STATUS_MAP };
+// const roleStatusMap = { ...ROLE_STATUS_MAP };
```

Comment:
_If this is an unused interface or variable, removing them would be better instead of commenting them out_